app/scripts/controllers/reviewRegistration.js

```diff
@@ -280,4 +329,117 @@ angular.module('confRegistrationWebApp')
     $scope.hasPendingCheckPayment = function(payments){
       return _.some(payments, { paymentType: 'CHECK', status: 'PENDING' });
     };
+
+    ////// START EVENT-433 //////
```

Comment:
I think it might be nice to put most of this in the spouse service and a registration service. Obviously anything interacting with the scope needs to stay. You should be able to return promises from the services and have a single `then` success/error handler for each operation. I know you inherited a lot of this code in one giant file so I'll let you make the decision as to what to do with it all. I think best practice is to usually put anything calling an api in a service.